packages/solid/src/__tests__/create_query_resource.spec.tsx

```diff
@@ -246,4 +247,37 @@ describe('createQueryResource', () => {
     await allPrevSettled(correctScope);
     await allPrevSettled(wrongScope);
   });
+
+  test('should update when query restarted', async () => {
+    const items = [1];
+
+    const query = createQuery({
+      handler: async (x: void) => [...items],
+    });
+
+    const mutation = createMutation({
+      handler: async (x: void) => items.push(2),
+    });
+
+    sample({
+      clock: mutation.finished.success,
+      target: query.start,
+    });
+
+    query.start();
```

Comment:
🤓 I'm more comfortable with tests for Fork API — it was created for testing, it allows you to start a Query in an isolated environment and track finishes of calculations. Could you rewrite this test with Fork API, please?